src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface Props {
+  pressedKey: string | null;
+}
+
+export class App extends React.Component {
+  state: Readonly<Props> = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
```

Comment:
It's extra, bc TypeScript already knows about React types
```suggestion
  componentDidMount() {
```